server/database/queries/order/getOrders.js

```diff
@@ -2,7 +2,7 @@ const connection = require('../../config/dbConnection');
 
 const getOrdersQuery = () => connection
   .query(
-    'SELECT orders.pk_i_id as key, orders.fk_i_place_id as storeId, orders.s_customer_address as address, orders.s_customer_phone as phone, orders.i_status as b_status, array( select json_build_array("pk_i_id","s_name","f_price") from items where items.fk_i_order_id = orders.pk_i_id) as items , sum(items.f_price) as price, orders.dt_create_at as date, tuser.s_name as captain FROM orders inner join places on places.pk_i_id = orders.fk_i_place_id inner join items on items.fk_i_order_id = orders.pk_i_id inner join TUser_order on TUser_order.order_id = orders.pk_i_id inner join tuser on tuser_order.tuser_id = tuser.pk_i_id where tuser.i_type = 1 group by orders.pk_i_id, TUser.s_name',
+    'SELECT orders.pk_i_id as key, orders.customer_name as customer, orders.fk_i_place_id as storeId, orders.s_customer_address as address, orders.s_customer_phone as phone, orders.i_status as b_status, array( select json_build_array("pk_i_id","s_name","f_price") from items where items.fk_i_order_id = orders.pk_i_id) as items , sum(items.f_price) as price, orders.dt_create_at as date, tuser.s_name as captain FROM orders inner join places on places.pk_i_id = orders.fk_i_place_id inner join items on items.fk_i_order_id = orders.pk_i_id inner join TUser_order on TUser_order.order_id = orders.pk_i_id inner join tuser on tuser_order.tuser_id = tuser.pk_i_id where tuser.i_type = 1 group by orders.pk_i_id, TUser.s_name',
   )
```

Comment:
you can use `COALESCE` function to  get captain and customer at once  and  use other types like left and right join or full join instead of inner join for example : 
```
select orders.pk_i_id as order_id, COALESCE(orders.customer_name, (select tuser.s_name from TUser inner join
tuser_order on tuser.pk_i_id = tuser_order.tuser_id where tuser.i_type = 2 and orders.pk_i_id = tuser_order.order_id))
 as customer_name, tuser.s_name as captain_name, tuser.pk_i_id from orders full join TUser_order on orders.pk_i_id
 = TUser_order.order_id inner join TUser on Tuser.pk_i_id=Tuser_order.tuser_id where tuser.i_type = 1;
```